recipes/use_cases/coding/llamas-in-code-review/README.md

```diff
@@ -0,0 +1,75 @@
+## Llamas in Code Review
+
+<video name="llama-code-review-loop" src="https://github.com/user-attachments/assets/f717889a-e517-4380-a07b-9657319dd189" controls></video>
+
+In this example, we have two agents:
+
+- **Code Author:** Writes the code.
+- **Code Reviewer:** Reviews the code and provides constructive feedback.
+
+Together, they'll engage in multiple iterations, and over time improve the code.
+
+This demo demonstrates tool calls, structured outputs and looping with llama.
+
+## Setup
+
+### Prerequisites
+
+- Python 3.10+
+- Docker
+
+### Running the demo
+
+We'll be using the fireworks llama-stack distribution to run this example - but you can use most other llama-stack distributions (instructions [here](https://llama-stack.readthedocs.io/en/latest/distributions/index.html)).
+(Though note that not all distributions support structured outputs yet e.g., Ollama).
```

Comment:
PR incoming for Ollama though: https://github.com/meta-llama/llama-stack/pull/680